src/lib.rs

```diff
@@ -450,13 +448,15 @@ pub mod prelude {
     #[doc(no_inline)]
     pub use crate::SubsecRound;
     #[doc(no_inline)]
+    pub use crate::{Date, Week};
```

Comment:
I don't think these need to be added to the prelude, since they're pretty niche.